README.md

```diff
@@ -222,6 +222,41 @@ When a pull request is approved and merged into main the deploy script will spin
 
 ![Architecture Diagram](./.images/architecture.svg?raw=true)
 
+```text
```

Comment:
This shouldn't be an individual text block, and the below items can probably be filtered into appropriate homes in the rest of the readme.